coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -3898,8 +3898,13 @@ def gelu(context, node):
     assert len(inputs) in (1, 2)
     if len(inputs) == 2:
         approximate = inputs[1].val
-        assert approximate == 'none'
-    res = mb.gelu(x=inputs[0], name=node.name)
+        if approximate == "tanh":
+            approximate = "TANH_APPROXIMATION"
+        elif approximate == "none":
```

Comment:
If pytorch only support two modes right now,
it will be better to do:
```
else:
    assert approximate == "none"
```
to deal with the future possible change in the torch frontend (like they add more support for different mode)

Reply:
I didn't do it because an unsupported approximation would still fail in the `mb.gelu` implementation. But you are right that this is a better place to signal where the conversion needs to happen, changing it now!